app/src/main/res/layout-land/add_profile_activity.xml

```diff
@@ -119,36 +127,53 @@
             android:visibility="@{viewModel.createPin ? View.VISIBLE : View.GONE}"
             app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_pin_check_box">
 
-            <org.oppia.android.app.profile.ProfileInputView
-              android:id="@+id/add_profile_activity_pin_profile_input_view"
+            <com.google.android.material.textfield.TextInputLayout
+              android:id="@+id/add_profile_activity_pin"
+              style="@style/TextInputLayout"
               android:layout_width="match_parent"
               android:layout_height="wrap_content"
               android:layout_marginStart="28dp"
               android:layout_marginTop="24dp"
               android:layout_marginEnd="28dp"
-              android:visibility="visible"
-              app:inputLength="3"
-              app:isPasswordInput="true"
-              app:label="@string/add_profile_input_pin"
+              app:errorMessage="@{viewModel.pinErrorMsg}"
               app:layout_constraintEnd_toEndOf="parent"
               app:layout_constraintStart_toStartOf="parent"
-              app:layout_constraintTop_toTopOf="parent"
-              profile:error="@{viewModel.pinErrorMsg}" />
+              app:layout_constraintTop_toTopOf="parent">
 
-            <org.oppia.android.app.profile.ProfileInputView
-              android:id="@+id/add_profile_activity_confirm_pin_profile_input_view"
+              <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/add_profile_activity_pin_edit_text"
+                style="@style/TextInputEditText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_profile_input_pin"
+                android:imeOptions="actionNext"
+                android:inputType="numberPassword"
+                android:maxLength="3" />
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <com.google.android.material.textfield.TextInputLayout
+              android:id="@+id/add_profile_activity_confirm_pin"
+              style="@style/TextInputLayout"
               android:layout_width="match_parent"
               android:layout_height="wrap_content"
               android:layout_marginStart="28dp"
-              android:layout_marginTop="8dp"
+              android:layout_marginTop="24dp"
```

Comment:
is the new margin according to the mocks?

Reply:
No, I'll revert that

Reply:
Done